doc/tokens/tokentypes/paper.rst

```diff
@@ -57,7 +57,7 @@ A good example for the ``token.enrolled.paper.top.html`` is:
 A good example for the ``token.enrolled.paper.bottom.html`` is:
 
     <p>
-      The paper token is a weak second factor. Please assure, that noone gets
+      The paper token is a weak second factor. Please assure, that no one gets
       hold  of this paper and can make a copy of it.
```

Comment:
double whitespace

Reply:
Do you mean between "hold  of"? 
I would like to not fix it at this point. This is nothing I added now and it is an HTML template, so ti does not matter.